### ISSUE TITLE
Receptor-ligand with gene names

### ISSUE THREAD

**Issue by mkrzak** [theislab/squidpy#291]
Hi folks,
Is there a way to specify column in var of anndata when running sq.gr.ligrec to use gene symbols given that my default gene names are ensemble ids? Or in other way around to use ensemble gene ids when calling sq.gr.ligrec.
Best, M

**Comment by giovp** [comment]
Hi @mkrzak ,

thanks for trying out Squidpy! For now this is not possible, although it shouldn't be difficult to add, we'd just have to check how it behaves when `use_raw=True`.

For now, this is what you could do:
```python
adata_reindex = adata.copy()
adata_reindex.var_names  = adata_reindex.var["new_gene_id"].values
sq.gr.ligrec(adata_reindex)
```

let us know if that works!

**Comment by mkrzak** [comment]
Hi @giovp ,
Thanks ! I am using this solution at the moment. Duplicated gene symbols are lost but cant think of a better way around it.
Cheers !

**Comment by giovp** [comment]
Right, ok. My understanding of the problem then is that we'd need to check whether omnipath provide the same amount of LR pairs in both gene ids correct? Because if the source data is in hgnc symbols, then it doesn't matter if you map ids, you'd still loose the duplicated ones? Also, out of curiosity, how many are they in the dataset you are currently looking at and are they biologically relevant?

**Comment by michalk8** [comment]
I haven't found a nice solution for duplicated gene symbols yet, but at least we can now easily specify them if they are present in `adata.var_names`.
closed via #395
